.add_event.php

```diff
@@ -2,10 +2,14 @@
 
 require_once ".db_config.php";
 require_once ".functions.php";
+require_once ".gcal.php";
 
 // empty response
 $response = null;
 
+// Google Calendar Object
+$calendar = new GoogleCalendar();
```

Comment:
I left a bunch of comments about this below. Sorry I'm going backwards on the review.